packages/component/src/Typography/Typography.base.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { memo } from 'react';
+import { Text } from 'react-native';
+import ThemeProvider from '@blue-learn/theme';
+import {
+	ColorTokens,
+	ColorTokensMap,
+	FontTokens,
+	TypographyBaseProps,
+	TypographyDecoration,
+	TypographyTransform,
+} from '@blue-learn/schema';
+
+/**
+ * Raw Component with Derived props + Theme
+ */
+
+const TypographyBase: React.FunctionComponent<
+	TypographyBaseProps
+> = ({
+	value = 'Lorem ipsum dolor sit amet.',
+	color = ColorTokens.Grey_70,
+	fontSize = FontTokens.sm,
+	fontWeight = 400,
+	textDecorationLine = TypographyDecoration.none,
+	textTransform = TypographyTransform.none,
+	fontFamily,
+	letterSpacing,
```

Comment:
add default value here too. but are we even using letterSpacing here. check with Vinit once on this. For now give default value and pass to text component

Reply:
okay

Reply:
done